Missfits Code 2018 PowerUp/src/org/usfirst/frc/team6418/robot/Robot.java

```diff
@@ -169,4 +339,46 @@ public void teleopPeriodic() {
 	@Override
 	public void testPeriodic() {
 	}
+	
+	public int MAKINGTHISAREDXcheckZone(boolean groundSwitchPressed, boolean switchSwitchPressed, boolean scaleSwitchPressed, 
+			boolean maxSwitchPressed, int currentZone, double elevatorSpeed) {
+		int zone;
```

Comment:
zone should be an [enum](http://tutorials.jenkov.com/java/enums.html) so that we have human-readable names rather than magic numbers.

Reply:
also, [naming](https://www.quora.com/Why-is-naming-things-hard-in-computer-science-and-how-can-it-can-be-made-easier)... if the "zone" we are in depends on the sign of elevatorSpeed, then it's not really a "zone" as much as a "mode" or "state"...